protocol-designer/src/components/StepEditForm/fields/TipPositionField/utils.ts

```diff
@@ -41,3 +45,70 @@ export function getDefaultMmFromBottom(args: {
       return DEFAULT_MM_TOUCH_TIP_OFFSET_FROM_TOP + wellDepthMm
   }
 }
+
+export const roundValue = (value: number | string | null): number => {
+  return round(Number(value), DECIMALS_ALLOWED)
```

Comment:
not sure what is the best value for `null`, but need to return a specific value when value is `null`

```suggestion
  return value === null ? 0 : round(Number(value), DECIMALS_ALLOWED)
```